### PR TITLE
Add class to forge button be easey to select for css

### PR DESCRIPTION
**Pull request by 6543**
just make it easy to switch icon via custom css

### CODE REVIEW

File: web/src/views/repo/RepoWrapper.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@
       <a v-if="badgeUrl" :href="badgeUrl" target="_blank">
         <img :src="badgeUrl" />
       </a>
-      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" />
+      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" class="icon-button forge" />
```
Comment (6543) [inline web/src/views/repo/RepoWrapper.vue:20]:
```suggestion
      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" class="forge" />
```

File: web/src/views/repo/RepoWrapper.vue [hunk 2]
```diff
@@ -17,7 +17,7 @@
       <a v-if="badgeUrl" :href="badgeUrl" target="_blank">
         <img :src="badgeUrl" />
       </a>
-      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" />
+      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" class="forge" />
```
Comment (anbraten) [inline web/src/views/repo/RepoWrapper.vue:20]: Do you want to use it for e2e test? In that case I would suggest

```suggestion
      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" data-test="forge" />
```

Comment (6543) [reply web/src/views/repo/RepoWrapper.vue:20]: No for custom styling on codeberg

**Comment by 6543** [comment]
Did test it already

-> visible: hidden and then add via background-image a new one ...

**Review by xoxys — APPROVED** [review]

**Pull request merged**